projects/copper-coreengine/src/main/java/org/copperengine/core/persistent/CommonSQLHelper.java

```diff
@@ -169,7 +169,7 @@ static StringBuilder appendSQLStates(StringBuilder sql, List<Object> params, Wor
         return result;
     }
 
-    public static List<AuditTrailInfo> processAuditResult(String sql, List<Object> params, Connection con, boolean loadMessage) throws SQLException {
```

Comment:
Again, keep the original method to avoid breaking changes:
```
@Deprecated
public static List<AuditTrailInfo> processAuditResult(String sql, List<Object> params, Connection con, boolean loadMessage) throws SQLException {
    return processAuditResult(params, con, loadMessage, true);
}

public static List<AuditTrailInfo> processAuditResult(String sql, List<Object> params, Connection con, boolean loadMessage, boolean clobSupported) throws SQLException {
    // your code here
}

Reply:
Keeping the public API stable makes sense. 
I will add the proposed changes with the next commit.